### DISCUSSION TITLE
LiveTranscription stuck in Pending status

### DISCUSSION THREAD

**Discussion by danish9878** [deepgram/community#391] [General help]
### Which Deepgram product are you using?

Deepgram SDKs

### Details

Hi,

I am using Deepgrams nodeJS SDK for live transcription. I tested the code in 
https://github.com/deepgram-devs/node-live-example and implemented the same in my application. Live transcription is working fine in the first go but in subsequent streaming requests, the audio transcription is stuck in pending status(checked the logs in usage).

How to reproduce the issue:
https://github.com/deepgram-devs/node-live-example -> When I run this code, the transcription works perfectly at first. But when i stop and restart the record button in UI, transcription won't work.

have attached the screenshot taken from usage logs and a few request IDs for reference

<img width="1274" alt="Screenshot 2023-10-30 at 1 30 04 PM" src="https://github.com/deepgram/community/assets/36759782/b6cf970d-cc3e-42dc-981c-22880a2dace5">

Status: Pending request ids

edc0d628-8ddb-47b1-8710-b76c7cae419e
1e910389-bf24-407c-93c5-20008a53bca8

### If you are making a request to the Deepgram API and have a request ID, please paste it below:

edc0d628-8ddb-47b1-8710-b76c7cae419e

**Comment by jpvajda** [comment]
@danish9878 👋  Would you be able to provide the code you are using? Have you modified the code at all from the [example provided here?](https://github.com/deepgram-devs/node-live-example ) If so it be helpful to see that code.

**Reply by danish9878** [reply]
@jpvajda Sure. We have separate backend and frontend servers running the respective code here. 

The backend implementation of [server.js](https://github.com/deepgram-devs/node-live-example/blob/main/server.js) code is exactly the same.

The client side implementation of [client.js](https://github.com/deepgram-devs/node-live-example/blob/main/public/client.js) has been modified.

```
// Voice recording
let microphone
const audioChunks = []
const audioElement = ref(null)
const audioSrc = ref(null)
const isRecording = ref(false)

const emit = defineEmits(['transcribe-audio', 'reset-transcript', 'send-transcript-message'])

defineProps({
  transcription: {
    type: String,
    default: ''
  }
})

async function getMicrophone() {
  const userMedia = await navigator.mediaDevices.getUserMedia({
    audio: true,
  });

  return new MediaRecorder(userMedia);
}

async function openMicrophone(microphone) {
  console.log('reset')
  emit('reset-transcript')
  await microphone.start(500);
  isRecording.value = true;

  microphone.onstop = () => {
    const blob = new Blob(audioChunks, { type: "audio/ogg; codecs=opus" });
    audioSrc.value = URL.createObjectURL(blob);
  };

  microphone.ondataavailable = (e) => {
    audioChunks.push(e.data);
    emit('transcribe-audio', e.data)
  };
}

async function closeMicrophone(microphone) {
  microphone.stop();
  isRecording.value = false;
}

async function recordAudio() {
  if (!microphone) {
    // open and close the microphone
    microphone = await getMicrophone();
    await openMicrophone(microphone);
  } else {
    await closeMicrophone(microphone);
    microphone = undefined;
  }
}

function sendTranscriptMessage() {
  emit('send-transcript-message')
}
```

**Comment by jpvajda** [comment]
We are discussing this issue in Discord. Closing as duplicate

https://discord.com/channels/1108042150941294664/1169570496828952586
